lib/schematype.js

```diff
@@ -824,8 +824,10 @@ SchemaType.prototype.get = function(fn) {
  *
  * From the examples above, you may have noticed that error messages support
  * basic templating. There are a few other template keywords besides `{PATH}`
- * and `{VALUE}` too. To find out more, details are available
- * [here](./error.html#Error.prototype.name).
+ * and `{VALUE}` too. They are:
```

Comment:
Move PATH and VALUE to the list, and replace last sentence with "Below is a list of supported template keywords"